plugins/amqp_compressed_proof_plugin/amqp_compressed_proof_plugin.cpp

```diff
@@ -328,6 +323,9 @@ void amqp_compressed_proof_plugin::plugin_initialize(const variables_map& option
       controller.accepted_block.connect([&generator=*my->proof_generator](const chain::block_state_ptr& p) {
          generator.on_accepted_block(p);
       });
+      controller.block_start.connect([&generator=*my->proof_generator](uint32_t block_num) {
```

Comment:
we should also wire up the signal in the unit test file

Reply:
actually you know.. that's just code duplication. probably what should be done is add a metohd  to `compressed_proof_generator` that takes a controller and wires it up.

Reply:
(I'll do that in the base PR)